string plan_path
---
duration time_elapsed
---
string goal_name
